### PR TITLE
simple_ingredients_page

### LINKED ISSUE #19: List Page Basic
For the basic list page, we need;

Add(or +) button to manually add a new list
(this doesn't have to be functional, purely placeholder is ok)

Selectable List of lists That triggers a pop up (list display)
(Can also be placeholder popup)

### PR DESCRIPTION
**Pull request by jaredkeeler2**
I want to update my simple_ingredients_page with the recent simple_lists_page code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +39,9 @@
     "react-native-safe-area-context": "4.10.5",
     "react-native-screens": "3.31.1",
     "react-native-web": "~0.19.10",
+    "expo-sqlite": "~14.0.6",
+    "expo-file-system": "~17.0.1",
+    "expo-asset": "~10.0.10"
```
Comment (yycorcino) [inline package.json]: Add a comma at the end.

File: app/(tabs)/lists.tsx [hunk 2]
```diff
@@ -1,35 +1,277 @@
+import React, { useState } from "react";
+import {
+  View,
+  Alert,
+  Modal,
+  Text,
+  Pressable,
+  StatusBar,
+} from "react-native";
+import DateTimePicker from "@react-native-community/datetimepicker";
```
Comment (yycorcino) [inline app/(tabs)/lists.tsx]: Commenting this out fixes 1 of the errors to make app load.

**Review by yycorcino — CHANGES_REQUESTED** [review]
You will be needing to address the comments.

For some reason, on my end I'm having database related issues that I'm seeing for the first time:

```
Warning: Encountered two children with the same key, `.$Chicken Pesto Wraps`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

**Comment by rjcpascual** [comment]
npm install
npm install react-native
npm install react-native-elements
npm install @react-native-community/datetimepicker
 prior to running program

**Review by yycorcino — APPROVED** [review]
Looks good!! With `package.json` updated and works go ahead and merge.

**Pull request merged**